level2/level2_Dohyung/src/Application.java

```diff
@@ -0,0 +1,61 @@
+
+import java.util.Scanner;
+
+public class Application {
+    public static void askQuestion(){
+        Scanner sc = new Scanner(System.in);
+
+
+        while(true){
+            try{
+                System.out.println("덧셈할 문자를 입력해주세요");
+                String inputString = sc.nextLine();
+                if((!inputString.contains(","))&&(!inputString.contains(":"))){ // 구분자로 ,나 :가 없을 시 커스텀구분자 메소드실행
+                    customCalculate(inputString);
+                }
+                calculate(inputString);
+            }catch(IllegalArgumentException e){
+                System.out.println("잘못된 값을 입력하셨습니다.");
+                break;
+            }
+        }
+    }
+    //기본 구분자 처리 메소드
```

Comment:
```suggestion
    }
    
    //기본 구분자 처리 메소드
```
줄바꿈 잘 해주는 것도 지켜야할 코드 컨벤션 중 하나입니다